src/algebra/discrete-log.md

```diff
@@ -6,10 +6,7 @@ The discrete logarithm is an integer $x$ solving the equation
 
 $$a^x \equiv b \pmod m,$$
```

Comment:
Replace `,` by `.`

Reply:
Can we add a line saying: "The discrete logarithm does not always exist, for instance there is no solution to $2^x \equiv 3 \pmod 7$. And there isn't a simple condition known that tells if the discrete logarithm exists or not."

Reply:
Sure, added.